config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   root 'homes#index'
   devise_for :users
+
+  resources :shop, only: [:index]
```

Comment:
will update :shop to :shops and resubmit PR

Reply:
updated in latest commit: commit feb18b1b55ca12546f189b4093df6ac70b145da2